openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -66,27 +66,27 @@ def _assert_success(self, data, expected_scopes):
         self.assertEqual(self.oauth2_adapter.get_client_for_token(token), self.oauth_client)
         self.assertEqual(self.oauth2_adapter.get_token_scope_names(token), expected_scopes)
 
-    def test_single_access_token(self):
+    @ddt.data(True, False)
+    def test_single_access_token(self, single_access_token):
         def extract_token(response):
             """
             Returns the access token from the response payload.
             """
             return json.loads(response.content)["access_token"]
 
         self._setup_provider_response(success=True)
-        for single_access_token in [True, False]:
```

Comment:
@nasthagiri Spent a little time looking at these failing tests. Reverting to the loop instead of the ddt decorator fixes the tests. I'm guessing there is some setUp or tearDown that happens between runs which is causing the ddt version to fail. Probably best to revert for now and move on.

Reply:
Ok. Thanks - will revert then.  I changed to `ddt` since they were failing earlier (but maybe just on my local devstack) and I wanted to know which ones exactly were failing.  I'll move forward with the revert!